package.json

```diff
@@ -16,6 +16,11 @@
     "openapi-generator": "^0.1.36",
     "ory-prettier-styles": "^1.0.1",
     "prettier": "2.0.4",
+    "text-runner": "5.0.0-pre9",
+    "textrun-make": "0.0.9",
+    "textrun-npm": "0.0.9",
+    "textrun-repo": "0.0.9",
+    "textrun-shell": "0.0.9",
```

Comment:
These npm packages are a form of "standard library" for Text-Runner and contain all the code block types used in this document. Everything is reusable elsewhere.